src/sentry/static/sentry/app/views/organizationEventsV2/events.tsx

```diff
@@ -18,6 +18,7 @@ import getDynamicText from 'app/utils/getDynamicText';
 import {getParams} from 'app/views/events/utils/getParams';
 
 import Table from './table';
+import FooTable from './foo_table';
```

Comment:
Is this a temporary name?

Reply:
@markstory Definitely temporary 😄  I'm not good with names.